internal/backends/android-activity/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+[package]
+name = "i-slint-backend-android-activity"
+description = "OpenGL rendering backend for Slint"
+authors.workspace = true
+edition.workspace = true
+homepage.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+[lib]
+path = "lib.rs"
+
+[features]
+game-activity = ["android-activity/game-activity"]
+native-activity = ["android-activity/native-activity"]
+
+[target.'cfg(target_os = "android")'.dependencies]
+# FIXME: we shoudln't depend on slint, but the other way around
+slint = { workspace = true, features = ["compat-1-2"] }
```

Comment:
I'm curious, why does the backend depend on Slint?

Reply:
I thought at first that the backend would be using public API only and so that would simplify things with regards to publishing it with a different version and a name like `slint-android 0.3.0`  or something.  But I guess this is futile.
Do you want me to go ahead and use the internal instead?

Reply:
I think that makes sense yeah. Either that or remove the FIXME :)